packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -469,16 +469,20 @@ export class ProjectActions {
     return { specs, specPattern, excludeSpecPattern, additionalIgnorePattern }
   }
 
+  setForceReconfigureProjectByTestingType (testingType: TestingType) {
+    this.ctx.update((coreData) => {
+      coreData.forceReconfigureProject = {
+        [testingType]: true,
```

Comment:
I'm not finding where this is reset to false. Should reconfigureProject set this to false after `this.ctx.actions.electron.showBrowserWindow()`?